.genie/wishes/_archive/pgserve-daemon-ownership/WISH.md

```
@@ -0,0 +1,140 @@
+  │   ├─ YES → read port from ~/.genie/pgserve.port → connect
+  │   └─ NO  → start daemon (which starts PG) → read port → connect
+  └─ Is connection healthy? (SELECT 1)
+      ├─ YES → proceed
+      └─ NO  → kill stale PG, restart daemon, retry once
+```
+
+No CLI command ever starts pgserve directly. If the daemon isn't running, the CLI auto-starts it. Self-healing on every connection attempt.
+
+## Changes
+
+### 1. Daemon advertises PG port (`scheduler-daemon.ts`)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)